test/screenplay/permanent_config_test.exs

```diff
@@ -19,6 +20,8 @@ defmodule Screenplay.PermanentConfigTest do
   end
 
   setup_all do
+    _ = start_supervised(Screenplay.Places.Cache)
```

Comment:
There is a `start_supervised!` if we don't care about the result (or rather, if we want to crash on failure instead of returning an `:error` that we then ignore).